### PR TITLE
Add avatar group

### PR DESCRIPTION
**Pull request by mouracamila**
This pull request adds the avatar group component

### CODE REVIEW

File: src/components/AvatarGroup/AvatarGroupCounter.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export type AvatarGroupdCounterProps = PropsWithChildren<{
+  total?: number;
+  href: string;
+}>;
+
+export const AvatarGroupCounter: React.FC<AvatarGroupdCounterProps> = ({ total, href }) => {
+  return (
+    <a
+      className="relative flex h-10 w-10 items-center justify-center rounded-full bg-gray-700 text-xs font-medium text-white ring-2 ring-gray-300 hover:bg-gray-600 dark:border-gray-800 "
```
Comment (rluders) [inline src/components/AvatarGroup/AvatarGroupCounter.tsx:11]: This color isn't right.

**Review by rluders — CHANGES_REQUESTED** [review]

File: src/components/AvatarGroup/AvatarGroupCounter.tsx (hunk 1 above)
Comment (mouracamila) [reply src/components/AvatarGroup/AvatarGroupCounter.tsx]: done

**Review by bacali95 — CHANGES_REQUESTED** [review]
Hey @mouracamila, why we don't have this component as `Avatar.Group` and also `Avatar.Counter` and locate it in the same folder as the `Avatar` one?

**Comment by mouracamila** [comment]
> Hey @mouracamila, why we don't have this component as `Avatar.Group` and also `Avatar.Counter` and locate it in the same folder as the `Avatar` one?

Hey, @bacali95  I just followed the button group and button style. :)  Do you want me to change it?

**Comment by bacali95** [comment]
> 
> 
> 
> 
> > Hey @mouracamila, why we don't have this component as `Avatar.Group` and also `Avatar.Counter` and locate it in the same folder as the `Avatar` one?
> 
> Hey, @bacali95  I just followed the button group and button style. :)  Do you want me to change it?
> 
> 

Yes please, it might be a good idea for me to change the button group as well 🤔

**Comment by bacali95** [comment]
LGTM

**Review by bacali95 — APPROVED** [review]

**Pull request merged**